src/program/services/filesystem/vfs/rivenvfs.py

```
@@ -115,6 +115,41 @@ def _acquire(self, host: str) -> pycurl.Curl:
+            c.setopt(pycurl.WRITEHEADER, header_buffer)
+
+            c.perform()
+
+            status_code = int(c.getinfo(pycurl.RESPONSE_CODE))
+
+            return status_code
+        finally:
+            try:
+                header_buffer.close()
+            except Exception:
+                pass
```

Ruff (0.13.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)